cmd/zoekt-indexserver/main.go

```diff
@@ -129,7 +129,7 @@ func periodicFetch(repoDir, indexDir string, opts *Options, pendingRepos chan<-
 // fetchGitRepo runs git-fetch, and returns true if there was an
 // update.
 func fetchGitRepo(dir string) bool {
-	cmd := exec.Command("git", "--git-dir", dir, "fetch", "origin")
+	cmd := exec.Command("git", "--git-dir", dir, "fetch", "origin", "--prune")
```

Comment:
did you test this? I'm guessing git is robust here, but normally flags come before the args so I would of expected.

```go
	cmd := exec.Command("git", "--git-dir", dir, "--prune", "fetch", "origin")
```